policies.md

```diff
@@ -40,7 +40,7 @@ As long as within our policies, our communities will manage themselves and are
 free to expand our rules with their own. Libera.Chat staff will not usually
 interfere in channel management and disputes unless network-wide policies are
 violated. We can offer to mediate or help out if our workload allows it, but
-we should not be the first line of defense.
+we should not be the first line of defence.
```

Comment:
This PR cannot be merged with this change present. All changes to policy, even typographical ones, require formal staff discussion and vote at our monthly general meeting. You're welcome to PR this change but it needs to be in a PR by itself and not combined with any other changes.

Reply:
That makes sense, I suppose!  Obviously, I'm not married to the change, so I'll pull it!  Thanks for the heads up.